Week 4/Enigma-java/src/main/java/Rotor.java

```diff
@@ -0,0 +1,46 @@
+import java.util.List;
+
+public class Rotor 
+{
+
+    public String rotorEncrypt(String msg, List<String> rotors)
+    {
+        final String alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        String encryptedMsg = "";
+        for (int i = 0; i < rotors.size(); i++)
+        {
+            //Swaps/resets variables after the first loop
+            if(i > 0)
+            {
+                msg = encryptedMsg;
+                encryptedMsg = "";
+            }
+
+            for(char letter: msg.toCharArray())
+            {
+                if(alphabet.indexOf(letter) == -1)
+                {
+
+                    encryptedMsg += letter;
+
+                }
+                else{
```

Comment:
## Error Handling 
If an unexpected character is entered by user, suitable error messages must be ensured to prevent the code from completely breaking